frameworks/Rust/may-minihttp/Cargo.toml

```diff
@@ -17,15 +17,18 @@ nanorand = { version = "0.7", default-features = false, features = ["std", "wyra
 buf-min = { version = "0.7", features = ["bytes"] }
 yarte = { version = "0.15", features = ["bytes-buf", "json"] }
 
-may = { version = "0.3.25", default-features = false }
+may = "0.3"
 may_postgres = { git = "https://github.com/Xudong-Huang/may_postgres.git" }
```

Comment:
This and the following line appear not to use a fixed version of the dependency - you should change them to refer to a specific commit ID.

Reply:
done